scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -605,6 +612,41 @@ s32 BPF_STRUCT_OPS(rusty_select_cpu, struct task_struct *p, s32 prev_cpu,
 			}
 		}
 	}
+	
+	/*
+	 * The task might also be forked/execd so check for wake up only
+	 */
+	if (wake_flags & SCX_WAKE_TTWU) {
+		u64 highest_vtime = 0;
+		s32 found_cpu = -1;
+		u64 current_vtime = p->scx.dsq_vtime;
+		struct cpu_task_data *ctd;
```

Comment:
nit: Could you please add a newline between the variable declarations and the actual code?